bindings/py/cpp_src/bindings/sdr/py_SDR.cpp

```diff
@@ -325,8 +325,11 @@ Argument dimensions A list of dimension sizes, defining the shape of the SDR.)",
 R"(See class nupic.bindings.sdr.Reshape)");
 
         py_SDR.def("flatten", [](SDR &self)
-            { return new Reshape(self, {self.size}); },
-R"(See class nupic.bindings.sdr.Reshape)");
+            {
+                auto flat = new SDR({ self.size });
+                flat->setSparse( self.getSparse() );
```

Comment:
is there c++ flatten to call here?

Reply:
Currently, C++ does not have a flatten method, C++ has reshape method only.